sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -343,4 +340,8 @@ object CommandUtils extends Logging {
       cs.copy(histogram = Some(histogram))
     }
   }
+
+  private def isDataPath(path: Path, stagingDir: String): Boolean = {
+    !path.getName.startsWith(stagingDir) && DataSourceUtils.isDataPath(path)
+  }
```

Comment:
Do we need to touch this file? This looks unnecessary refactoring.

Reply:
Ur, is this used somewhere else?

Reply:
Yes. change `DataSourceUtils.isDataPath(path)` to `DataSourceUtils.isDataPath(status.getPath)`.

Reply:
Got it. Thanks